tests/integration/targets/module_helper/tasks/msimple_output_conflict.yml

```diff
@@ -29,3 +29,22 @@
       - simple1 is not changed
       - simple1.value is none
       - 'simple2._msg == "a message in a bottle"'
+      - 'simple2.msg == "a message in a bottle"'
```

Comment:
Is it a good idea to have both `msg` and `_msg` returned?

Reply:
The `_msg` version is kept for compatibility (TBF it wasn't working until #5755 ).

Then, when handling exceptions, MH will overwrite `msg` with its own content, so in that case it is good to have both so we do not lose the original content entirely.

Reply:
I'm wondering whether it is a good idea to make this toggle-able. Changing opkg to now return both `_msg` and `msg` is also somewhat ugly.